packages/test_common/src/mocks/erc721_enumerable.cairo

```diff
@@ -0,0 +1,154 @@
+#[starknet::contract]
+pub mod ERC721EnumerableMock {
```

Comment:
Don't we have the same mock in `mocks/erc721.cairo`? Why not adding the extra impl there instead of creating a new file?

Reply:
Removed the duplicate